lib/morandi/profiled_pixbuf.rb

```diff
@@ -38,14 +38,29 @@ def initialize(*args)
       end
     end
 
-    super(*args)
-  rescue Gdk::PixbufError::CorruptImage => e
+    # TODO: This is to fix some deprecation warnings. This needs refactoring.
+    # All can be implemented without having to hack on the PixBuff gem.
+    case args.size
+    when 1
+      super(file: args.first)
+    when 3
+      super(path: args[0], width: args[1], height: args[2])
+    else
+      super(*args)
```

Comment:
presumably if this line is ever exercised it will hit the same deprecation issues that we're encountering currently?

Reply:
there will be more deprecations, yes
I don't think I fixed them all. 
The problem is that we should use GdkPixbuff without trying to overwrite it. It needs more extensive refactoring. 
This is Good enough for now and as a start base :)